src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action = " " 
+      method = "get" 
+      data-qa="big"
+      class = "big_search_bar"
+    >
+      <label>
+        <div class = "search_bar">
+          <input 
+          type = "text" 
+          name = "big_search_bar"
+          data-qa ="keypress"
+          placeholder = "Try &quot;Los Angeles&quot;"
+          class = "input big_input"
+          >
+        </div>
+      </label>
```

Comment:
```suggestion
    <form
      action="#" 
      method="get" 
      data-qa="big"
      class="big_search_bar"
    >
      <label>
        <div class="search_bar">
          <input 
            type="text" 
            name="big_search_bar"
            data-qa="keypress"
            placeholder="Try &quot;Los Angeles&quot;"
            class="input big_input"
          >
        </div>
      </label>
```
fix formatting
do not leave spaces in attributes class = "" (BAD) class="" (GOOD)

Reply:
fix bellow as well